src/components/SectionTitle/SectionTitle.style.ts

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components'
+import { colors } from 'styles/colors'
+
+import { breakpoints } from 'shared/breakpoints'
+
+interface ISectionTitle {
+  size: 'sm' | 'md' | 'lg' | 'xl'
+}
+
+const TitleSizeStyles = (size: string) => {
+  switch (size) {
+    case 'sm':
+      return `
+        font-size: 2rem;
+      `
+    case 'md':
+      return `
+          font-size: 3rem;
+        `
+    case 'lg':
+      return `
+        font-size: 5rem;
+      `
+    case 'xl':
+      return `
+        font-size: 6rem;
+      `
+    default:
+      break
+  }
+}
+
+export const Title = styled.h2<ISectionTitle>`
+  display: inline;
+  font-style: normal;
+  font-size: 3rem;
```

Comment:
The 3rem font size is not necessary because you pass the size in props.

Reply:
done